tests/categorical_test.py

```diff
@@ -106,4 +106,17 @@ def testBinop(self):
         
         with self.assertRaises(NotImplementedError):
             cat._binop('string 1', '===')
-        
\ No newline at end of file
+            
+    def testConcatenate(self):
+        catOne = self._getCategorical()
+        catTwo = self._getRandomizedCategorical()
+        
+        resultCat = catOne.concatenate([catTwo])
```

Comment:
I'd like to make sure we also test the preferred idiom of `resultCat = ak.concatenate([catOne, catTwo])`, either by changing this line or adding a separate test (even better).

I *think* `ak.concatenate(..., ordered=False)` should also work with Categoricals, but we should explicitly test that one too.

Reply:
@reuster986 Done! Just pushed the commit including the two tests you requested.

Reply:
@reuster986 Please review when you get a chance and lemme know if you have any other changes you'd like to see.